samples/kubernetes/end2end/docs/08-alert.md

```diff
@@ -0,0 +1,81 @@
+## Firing Alerts
+In this step we show you how to fire alerts.
+
+### Configuring Alerts
+We do all the alerting related configurations in the customized [values.yaml](../prometheus/values.yaml) when installing the Prometheus chart.  
+
+First, configure alerting rules. For demonstration purpse, We define one [alerting rule](../prometheus/values.yaml#L45) named 'ClusterWarning' that will fire alerts if some WLS cluster has only one running server.
+
+Second, configure Alertmanager. In [alertmanager.yml](../prometheus/values.yaml#L27) we define one route and one receiver that will send all the received alerts to a webhook.
+
+### Verifying Configuration
+Access the Prometheus alerts page in your browser with the URL `http://<HostIP>:30000/alerts`. Check the configured alert rules. You should get something like below.  
+
+![Alert Rule](./images/alert-rule.png)  
+
+Go to Alert Manager web UI via the URL `http://<HostIP>:32000`. 
+- Check the configured rule and receiver via the URL `http://<HostIP>:32000/#/status`. You should get something similar as below.  
+
+![Alert Manager Configuration](./images/alert-manager-config.png)  
+- Check the alert page via the URL `http://<HostIP>:32000/#/alerts`. There is no received alert yet.
+
+### Triggering Alerts
+To trigger alerts, one WLS cluster need to have only one managed server running. 
+- Update file `demo-domains/domain1.yaml` to change the line from `replicas: 2` to `replicas: 1`.
+- Apply the updated domain resource file.
+  ```
+  kubectl apply -f demo-domains/domain1.yaml
+  ```
+If the new domain resource is applied successfully, one WLS server will start to termiate. Wait until only one managed server is running.
```

Comment:
one of the WebLogic servers is terminated. Wait until only one managed server is running in the cluster.